### PR TITLE
Add linux board (OBAL)

### PR DESCRIPTION
**Pull request by HefnySco**
Adding Open Board Architecture for Linux  (Obal)  hardware board.
Obal is merged in [PR 18537](https://github.com/ArduPilot/ardupilot/pull/18537)

### CODE REVIEW

**Comment by Hwurzburg** [comment]
this is missing the actual page

**Comment by HefnySco** [comment]
> this is missing the actual page
My mistake :(

**Comment by Hwurzburg** [comment]
thanks, where does one buy this board?

**Comment by HefnySco** [comment]
It is open source and free.
The PCB can be fabricated for 5$ and other component can be soldered as
breakouts. This is the idea behind it.... A simple completely open source
easy to build AP board.
The link to PCB https://oshwlab.com/mohammad.hefny/Ardupilot-OBAL-FCB

My target is to provide non-profit DIY board.

**Comment by Hwurzburg** [comment]
great, but unfortunately, the link to purchase the board and components does not seem to be working at the moment...for this to be more than a discuss.ardupilot.org blog post and an actual autopilot page, a way to purchase the kit  has to be available

**Comment by HefnySco** [comment]
@Hwurzburg I added links to purchase the items - and you can purchase them from many other sellers-. The board gerber file is what you need to send to any PCB to make the board.

**Comment by brunoolivieri** [comment]
First of all, thank you!  I am anxious to test/use that!

Did you miss the LED & Buzzer Driver (ULN2803APG) link?

Is the manual in wiki already?

**Comment by HefnySco** [comment]
> Is the manual in wiki already?

@brunoolivieri 

I created this Help [https://github.com/HefnySco/OBAL](https://github.com/HefnySco/OBAL) If it is not enough please let me know.

**Comment by brunoolivieri** [comment]
Thanks @HefnySco, LGTM

**Comment by HefnySco** [comment]
@Hwurzburg please advise if latest updated were enough. The links contains everything user need to buy including how to get the board.

File: common/source/docs/common-autopilots.rst [hunk 1]
```diff
@@ -35,6 +35,7 @@ Open hardware
     OpenPilot Revolution <common-openpilot-revo-mini>
     PocketPilot* (Linux) <https://github.com/PocketPilot/PocketPilot>
     TauLabs Sparky2** <common-taulabs-sparky2>
+    Obal Board (Linux) <common-obal-overview>
```
Comment (Hwurzburg) [inline common/source/docs/common-autopilots.rst:38]: couple of final nits....should be placed in list in alphabetical order

File: common/source/docs/common-obal-overview.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
+.. _common-obal-overview:
+
+=========================
```
Comment (Hwurzburg) [inline common/source/docs/common-obal-overview.rst:3]: these must be the same length as the text below

File: common/source/docs/common-obal-overview.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Open Board Architecture for Linux - OBAL Overview
+=========================
```
Comment (Hwurzburg) [inline common/source/docs/common-obal-overview.rst:5]: these must be the same length as the text above

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
good to go after these changes..thanks

**Comment by HefnySco** [comment]
@Hwurzburg Done ... thank you for your detailed feedback.

**Comment by Hwurzburg** [comment]
thanks...merging

**Pull request merged**